File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        a = {}
        for num in nums:
            a[num] = a.get(num, 0) + 1
        
        res = sorted(a.keys(), key=lambda x: a[x], reverse=True)
        return res[:k]

File: Data Structures & Algorithms/number-of-one-bits/submission-1.java
class Solution {
          int res = 0;
    public int hammingWeight(int n) {
  
        if(n == 0){
            return res;
        }
        if(n%2 == 1){
            res+= 1;
        }
        return hammingWeight(n/2);
    }
}
